concordium-consensus/src/Concordium/Scheduler/ProtocolLevelTokens/Queries.hs

```diff
@@ -129,7 +129,7 @@ queryTokenInfo tokenId bs = case sSupportsPLT (accountVersion @(AccountVersionFo
                                       tsTotalSupply = toTokenAmount _pltDecimals totalSupply,
                                       tsModuleState = tms
                                     }
-                        return $ Right TokenInfo{tiTokenId = tokenId, tiTokenState = ts}
+                        return $ Right TokenInfo{tiTokenId = toUpperCase tokenId, tiTokenState = ts}
```

Comment:
```suggestion
                        return $ Right TokenInfo{tiTokenId = _pltTokenId, tiTokenState = ts}
```
The canonical form is what's specified in `PLTConfiguration`. Not converted to uppercase.